deepl-connector-test/src_test/com/axonivy/connector/deepl/test/DeepLTest.java

```diff
@@ -22,7 +22,7 @@
 import ch.ivyteam.ivy.rest.client.security.CsrfHeaderFeature;
 import ch.ivyteam.ivy.scripting.objects.File;
 
-@IvyProcessTest
+@IvyProcessTest(enableWebServer = true)
```

Comment:
important roll-out; if we don't flag tests requiring the REST stack with `enableWebServer`, the build fails from 10.0.13 on-wards.
it derives from a performance-improvement, we no longer start the heavy http stack, for all tests that can live without it